modules/tinymce/src/core/main/ts/commands/RangeUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { Fun, Optional } from '@ephox/katamari';
```

Comment:
This is missing the license header file. Also this doesn't seem specific to commands, so should this be in the `selection` or similar directory instead?

- [ ] Add missing license header file
- [ ] Consider moving this to another location, or merge it with one of the existing modules in `selection`.

Reply:
Also should we have tests for this specific module, or are we happy with the Font/Line Height commands testing this enough?

Reply:
I really want to be lazy say that we should trust in the current tests for the Font/Line Height commands. However, if you (or anyone else) think it needs its own tests I'll write them.